examples/python/find_coauthored_papers/README.md

```diff
@@ -0,0 +1,40 @@
+Given two author names, this script finds coauthored papers.
+In the case that multiple authors match either author name,
+a summary of each author is printed. The user should then re-run
```

Comment:
Nit: "a summary of each match is printed" just feeling like we use the word author to refer to the user input already so confusing to use it again.